Marlin/temperature.cpp

```diff
@@ -512,24 +513,27 @@ inline void _temp_error(int e, const char *serial_msg, const char *lcd_msg) {
 
 void max_temp_error(uint8_t e) {
 // Temp error has been reset
-  if (Cartridge__GetPresentCheck()){
+  if (Cartridge__GetPresentCheck()) {
     if (time_since_last_err[e] == 0) {
       time_since_last_err[e] = millis();
     }
   // There has been a recent error, if was more than a second ago, it is probably an error
     else if (millis() > time_since_last_err[e] + TEMP_ERROR_INTERVAL) {
-      if(!Cartridge__FFFNotPresentHysteresis()) 
+      if(Cartridge__Present(e)) 
```

Comment:
Good point. The change I made is not equivalent to the original code as I initially thought.

As we discussed, setting time_since_last_err[e] = millis() when the cartridge is inserted might be a way to avoid triggering this error, but still maintain the modularity of the original cartridge insertion code.

The reason it is in this branch is because we need the same temperature/cartridge handling now that we have enabled the thermistor in T1 for heated build chamber. I COULD separate that into it's own branch... maybe...